rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+---
+created-on: 2021-03-23   (date the RFC was created)
```

Comment:
(Delete the stuff in ()'s I put in in as a comment should have noted to delete them, heh)

Reply:
will do